zuko-ui/src/components/Article/Article.jsx

```diff
@@ -11,7 +11,7 @@ function Article({data = {}}){
             {data?.categories ? 
             <div className={styles.categories}>
                 {data?.categories.map((el, i) => (
-                    <span className={styles.subtitle}>{el?.name}</span>
+                    <span className={styles.subtitle} key={`${el}-${i}`}>{el?.name}</span>
```

Comment:
No need for the full object you can just give it the id of the object so el.id as that would be unique for every item